x-pack/plugins/aiops/common/constants.ts

```diff
@@ -25,6 +25,11 @@ export const RANDOM_SAMPLER_SEED = 3867412;
 export const CASES_ATTACHMENT_CHANGE_POINT_CHART = 'aiopsChangePointChart';
 
 export const EMBEDDABLE_CHANGE_POINT_CHART_TYPE = 'aiopsChangePointChart' as const;
+export const EMBEDDABLE_CHANGE_POINT_TABLE_TYPE = 'aiopsChangePointTable' as const;
+
+export type EmbeddableChangePointType =
```

Comment:
As discussed, can these be ordered alphabetically? Or use a single 'Change point' item and allow the user to select whether they want charts or the table.

<img width="251" alt="image" src="https://github.com/elastic/kibana/assets/7405507/d8e746aa-8596-43d4-9014-e675e8528ada">

Reply:
Using the single 'Change point' option could also be followed in the ML page, allowing the user to select whether they want to add as a table or as charts.

Reply:
<img width="259" alt="image" src="https://github.com/elastic/kibana/assets/5236598/641312de-c5d8-424a-8da9-d2f6291255f0">

Decided to use a single embeddable type with an additional "View type" parameter